packages/terra-icon/package.json

```diff
@@ -23,13 +23,13 @@
   "devDependencies": {
     "csvtojson": "^1.1.4",
     "jsdom": "^9.11.0",
+    "cerner-one-icons": "git+https://github.com/cerner/one-cerner-style-icons.git#2e0f04a572e828aff3f1f39dfcb1c496fedd46c6",
```

Comment:
Lets update this to point to a release tag or tarball file so we have a better since of what version we are consuming.